pages/common/csvpy.md

```diff
@@ -0,0 +1,12 @@
+# csvpy
+
+> Loads a CSV file into a Python shell.
+> Included in csvkit.
+
+- Load a CSV file into a _CSVKitReader_ object:
```

Comment:
That's Markdown. I concur. Generally we tend to use backticks for formatting or referring to tokens or the like @oxguy3